plugin/ale/cursor.vim

```diff
@@ -72,7 +72,7 @@ function! ale#cursor#TruncatedEcho(message)
 
     try
         " Echo the message truncated to fit without creating a prompt.
-        set shortmess+=T
+        setl shortmess+=T
```

Comment:
`setlocal` would be better here. do you need to also change the `let` below, or the value of the variable above?

You should probably write the full name of `setlocal` here. It might be easy to miss the `l`. Though we do use `exec` a lot instead of `execute`, etc.